control_panel_api/tests/test_views.py

```diff
@@ -103,17 +107,18 @@ def test_delete(self, mock_aws_delete_role):
             reverse('user-detail', (self.fixture.auth0_id,)))
         self.assertEqual(HTTP_404_NOT_FOUND, response.status_code)
 
-    @patch('control_panel_api.models.User.helm_create')
     @patch('control_panel_api.models.User.aws_create_role')
-    def test_create(self, mock_aws_create_role, mock_helm_create_user):
-        data = {'auth0_id': 'github|2', 'username': 'foo'}
+    def test_create(self, mock_aws_create_role):
+        username = 'foo'
+        data = {'auth0_id': 'github|2', 'username': username}
         response = self.client.post(reverse('user-list'), data)
         self.assertEqual(HTTP_201_CREATED, response.status_code)
 
         self.assertEqual(data['auth0_id'], response.data['auth0_id'])
 
         mock_aws_create_role.assert_called()
-        mock_helm_create_user.assert_called()
```

Comment:
The view layer tests are asserting that model methods are called (which have no params) so need to remain.  The model layer tests should be asserting the correct params.

Reply:
True, I may have gone overboard with the test updates. I'll revert.